text/0034-telemetry-policies.md

```diff
@@ -425,6 +426,23 @@ This property allows us to gather essential information to better understand on
 | filtered_by_before_finished_at        | `true` if tasks are filtered by the `beforeFinishedAt` query parameter, otherwise `false` | false |
 | filtered_by_after_finished_at         | `true` if tasks are filtered by the `afterFinishedAt` query parameter, otherwise `false` | false |
 
+## `Tasks Deleted`
+
+| Property name | Description | Example |
+|---------------|-------------|---------|
+| user_agent    | Represents the user-agent encountered on this call. | `["Meilisearch Ruby (v2.1)", "Ruby (3.0)"]` |
+| filtered_by_uid                       | `true` if tasks are filtered by the `uid` query parameter, otherwise `false` | false |
+| filtered_by_index_uid                 | `true` if tasks are filtered by the `indexUid` query parameter, otherwise `false` | false |
+| filtered_by_type                      | `true` if tasks are filtered by the `type` query parameter, otherwise `false` | false |
+| filtered_by_status                    | `true` if tasks are filtered by the `status` query parameter, otherwise `false` | false |
```

Comment:
```suggestion
| filtered_by_uid                       | `true` if tasks are filtered by the `uids` query parameter, otherwise `false` | false |
| filtered_by_index_uid                 | `true` if tasks are filtered by the `indexUids` query parameter, otherwise `false` | false |
| filtered_by_type                      | `true` if tasks are filtered by the `types` query parameter, otherwise `false` | false |
| filtered_by_status                    | `true` if tasks are filtered by the `statuses` query parameter, otherwise `false` | false |
```
Shouldn't these be plurals? Also, shouldn't the "Property names" be updated? E.g. `filtered_by_uid`-> `filtered_by_uids`

Reply:
From my POV, it means tasks have been filtered by their uid field, so it's ok!